.github/actions/generate-test-dbs/action.yml

```diff
@@ -0,0 +1,27 @@
+name: "generate-test-dbs"
+description: "Generate qcodes dbs used for upgrade tests"
+runs:
+  using: "composite"
+  steps:
+    - name: generate test dbs
+      run: |
+        pip install -e .
+        pwd
+        cd ..
+        git clone https://github.com/QCoDeS/qcodes_generate_test_db.git
+        cd qcodes_generate_test_db
+        python generate_version_0.py
+        python generate_version_1.py
+        python generate_version_2.py
+        python generate_version_3.py
+        python generate_version_4a.py
+        python generate_version_4.py
+        python generate_version_5.py
+        python generate_version_6.py
+        python generate_version_7.py
+        python generate_version_8.py
```

Comment:
is it worth the effort to use `actions/cache@v2` to maintain a cache of DB files as well?

Reply:
Might savew 40 sec or so. I would say its a nice upgrade but can be done separately.